css/css-basics_social-card/css/styles.css

```diff
@@ -1,3 +1,22 @@
 * {
   box-sizing: border-box;
 }
```

Comment:
The body must have a different background-color than the article so that there is a visible difference. When the article is white it has to be something like grey or smokewhite.

Reply:
when i looked at it it had a nice visible difference